draft-ietf-quic-multipath.md

```diff
@@ -448,6 +456,13 @@ However, implicit signals such as idle time or packet losses might be
 the only way for an endhost to detect path closure (see
 {{idle-time-close}}).
 
+PATH_ABANDON frame causes all CIDs allocated by both
+of the endpoints for the specified Path ID to be retired.
```

Comment:
Actually I'm not sure this sentence is correct. You still need to send a CID_RETIRE frame to actually retire the CIDs, no?

Reply:
This sentence try to explain that PATH_ABANDON trigger the CID retirement to start. 
Basically endpoints still need to send CID_RETIRE frame, but this action itself is caused by PATH_ABANDON.

Reply:
How about this:
```suggestion
When a path is abandoned, all CIDs allocated by both
of the endpoints for the specified Path ID need to be retired.
```

Reply:
However, I'm not sure if it would be better to remove this sentence. Path abandoned should be discussed in the next subsection and not here. However, I don't think this sentence I needed in the next subsection as this is already explained normatively in more detail.

Reply:
No, the previous writing is better. We would like the abandon path to just drop all the connection ID allocated for the path, without requiring explicit "retire connection ID" per path.

But this clearly needs more discussion.